ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1448,6 +1448,9 @@ IE_GPU.matmul_2x2_2x2
 IE_GPU.matmul_2x3_3x3
 IE_GPU.matmul_3x2_3x3_transpose
 IE_GPU.matmul_3x2_2x3_transpose
+IE_GPU.matmul_2_transpose_2x1x2
+IE_GPU.matmul_2x2x3_3
+IE_GPU.matmul_2x3x2_2_transpose
```

Comment:
Is the rest of the tests disabled in some other place? e.g. matmul_2x0_0x2

Reply:
No, they are not disabled, I have excluded only the ones introduced with this commit.